### ISSUE TITLE
Routes need to be defined without leading `/`

### ISSUE THREAD

**Issue by vishy1618** [FormidableLabs/redux-little-router#29]
The routes as defined on the home-page doesn't work:

```javascript
const routes = {
  '/': {
    title: 'Home'
  },
  '/messages': {
    title: 'Message'
  },
  '/messages/:user': {
    title: 'Message History'
  }
};
```

You need to define them without the leading slash:

```javascript
const routes = {
  '/': {
    title: 'Home'
  },
  'messages': {
    title: 'Message'
  },
  'messages/:user': {
    title: 'Message History'
  }
};
```

This issue was observed after upgrading to the newest version (7.0.0).

**Comment by vishy1618** [comment]
FWIW, they work if I define the optional `basename` parameter as `""`, a blank string.

**Comment by exogen** [comment]
Same as #27?

**Comment by tptee** [comment]
Yep!

@vishy1618 I'll be fixing this today!

**Comment by tptee** [comment]
@vishy1618 Fixed in `7.0.1` 🎉

**Comment by david-davidson** [comment]
🎉
